### PR TITLE
add markdown file to collate results from gem5art experiments

### PR DESCRIPTION
**Pull request by tv-reddy**

### CODE REVIEW

File: others/status-plots.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+![Boot Tests Status with Classic Memory and systemd Boot](status-plots/boot_classic_systemd.png)
+
+![Boot Tests Status with MI_Example Memory and init Boot](status-plots/boot_MI_example_init.png)
+
+![Boot Tests Status with MI_Example Memory and systemd Boot](status-plots/boot_MI_example_systemd.png)
+
+![Boot Tests Status with MESI_Two_Level Memory and init Boot](status-plots/boot_MESI_Two_Level_init.png)
+
+![Boot Tests Status with MESI_Two_Level Memory and systemd Boot](status-plots/boot_MESI_Two_Level_systemd.png)
+
```
Comment (powerjg) [inline others/status-plots.md:29]: Can we have a summary here?

X, Y, and Z works fine. There seem to be bugs with W. We're not sure what's going on with A and B.

File: others/status-plots.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+# GAPBS Tests
+
+# SPEC 2006 Tests
+
+The following plot represent the status of SPEC2006 workloads for different CPUs and data sizes with respect to gem5-20, linux kernel version 4.19.83 and gcc version 7.5.0.
+
+![SPEC-2006 status fro gem5-20 ](status-plots/spec2006_gem5-20_status.png)
+
+* Few workloads, [**400.perlbench**, **447.dealII**, **450.soplex**, **483.xalancbmk**] had build errors
```
Comment (powerjg) [inline others/status-plots.md:57]: Could you add these to the plot so you can visually tell there's errors?

File: others/status-plots.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+* **434.zeusmp** had crashed in gem5-19 as well due a segmentation fault.
```
Comment (powerjg) [inline others/status-plots.md:58]:
```suggestion
* **434.zeusmp** had crashed in gem5-19 and gem5-20 segmentation fault.
```

File: others/status-plots.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+It is a known problem that if the number of simulated CPU cores increase, KVM simulations get stuck sometimes.
+A work around is to use lower number of event queues than the CPU cores.
+Although our scripts do that for more than 1 CPU core, the cases shown as `timeout` in the plot above
+suffer from this problem of getting stuck.
+
+![NPB Status with TimingSimple CPU](status-plots/npb_multicore_timing.png)
+
+There are three cases with TimingSimple CPU which did not finish in the alloted time.
+There is no reason apparent in the generated results files (`simout`, `simerr`, `system.pc.com_1.device`).
+
```
Comment (powerjg) [inline others/status-plots.md:46]: Can we have a summary?

File: others/status-plots.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+These NPB tests use KVM CPU (1,8,16,32, and 64 cores) and TimingSimple CPU (1 and 8 cores) with MESI_Two_Level memory system.
+
+![NPB Status with KVM CPU](status-plots/npb_multicore_kvm.png)
+
+It is a known problem that if the number of simulated CPU cores increase, KVM simulations get stuck sometimes.
+A work around is to use lower number of event queues than the CPU cores.
+Although our scripts do that for more than 1 CPU core, the cases shown as `timeout` in the plot above
+suffer from this problem of getting stuck.
+
+![NPB Status with TimingSimple CPU](status-plots/npb_multicore_timing.png)
```
Comment (powerjg) [inline others/status-plots.md:42]: What was the coherence protocol/memory system?

File: others/status-plots.md (hunk 1 above)
Comment (ayaz91) [reply others/status-plots.md:33]: Done

File: others/status-plots.md (hunk 4 above)
Comment (ayaz91) [reply others/status-plots.md:67]: Done

File: others/status-plots.md (hunk 5 above)
Comment (ayaz91) [reply others/status-plots.md]: updated

**Comment by ayaz91** [comment]
Hey everyone,
I have pushed this change to Gerrit now. Please, directly push any new changes there.

**Comment by ayaz91** [comment]
Closing this as It was directly pushed to Gerrit

**Pull request closed without merging**